packages/node/src/integrations/http.ts

```diff
@@ -23,6 +24,8 @@ import { getRequestUrl } from '../utils/getRequestUrl';
 
 const INTEGRATION_NAME = 'Http';
 
+const INSTRUMENTATION_NAME = '@opentelemetry/instrumentation-http-sentry';
```

Comment:
I feel like prepending with `@opentelemetry` can be confusing because users might ignore `-sentry` at the end. Maybe we prepend it with `sentry-patched` or something similar to make it clear to users about how it works?

Reply:
```suggestion
const INSTRUMENTATION_NAME = '@opentelemetry_sentry-patched/instrumentation-http';
```

what about this?

Reply:
that feels better to me, let's go with that.